temp/readme.md

```diff
@@ -0,0 +1,63 @@
+
+## Headers H1-H6 
+
+# H1 Header 
+
+## H2 Header 
+
+### H3 Header 
+
+#### H4 Header 
+
+##### H5 Header 
+
+###### H6 Header 
+
+## Lists Ordered, Unordered sub-list, Unordered 
+
+
+1. One
+2. Two
+3. Three
+4. Four
+5. Five
+ 
+
+
+1. One
+  * Two
+4. Three
+ 
+
+
+* Five
+* Four
+* Three
+* Two
+* One
+ 
+
+## Links 
+
+[Link One Google](google.com)
+
+[Link Two Yahoo](yahoo.com)
+
+[Link Three Reddit](reddit.com)
+
+## Code Blocks 
+
+```js
+<h1>H1 Title</h1>
+<p>This is a code block filler text.</p>
+```
+
+```js
+<h2>H2 Title</h2>
```

Comment:
Output logic flaw: html is not javascript

Reply:
Thanks for the feedback ill make the changes.